modules/local/generate_summary.nf

```diff
@@ -0,0 +1,34 @@
+process GENERATE_SUMMARY {
+    tag "$meta.id"
+    label 'process_single'
+
+    if (params.enable_conda) {
+        exit 1, "Conda environments cannot be used when using the GENERATE_SUMMARY module. Please use docker or singularity containers."
+    }
+    container 'genomehubs/blobtoolkit:4.0.7'
+
+    input:
+    tuple val(meta), path(blobdir)
+    tuple val(meta), path(png)
+
+    output:
+    tuple val(meta), path("${blobdir}/summary.json.gz") , emit: json
+    path "versions.yml"                              , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def args = task.ext.args ?: ''
+    def output = "${blobdir}/summary.json.gz"
```

Comment:
Like above, does the output file really have to be added to the input directory ?